src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.html

```diff
@@ -0,0 +1,136 @@
+<div class="detail-container" *ngIf="workItem" id="workItemDetail_Wrapper">
+  <form *ngIf="active" 
+        (ngSubmit)="save()"          
+        #almWorkItemDetailForm="ngForm" 
+        id="wi-detail-form">
+    <fieldset [attr.disabled]="!loggedIn ? true : null">
+      <div class="width100 pull-left mobile-bar paddingL20 paddingR20">
+        <div class="pull-left dropdown-kebab-pf detail-type-dropdown" dropdown>
+          <span dropdown-open class="wi-type-icon" almIcon [iconType]="workItem.type"></span>
+          <ul class="dropdown-menu mobMarginL20" aria-labelledby="dropdownKebabRight">
+            <li *ngFor="let type of workItemTypes" (click)="onChangeType(type.name)">
+              <a>
+                <span class="wi-type-icon dropdown-icon" almIcon [iconType]="type.name"></span>
+                <span class="dropdown-text">
+                  {{type.name.substring(7,type.length)}}
```

Comment:
@joshuawilson Actually once we have JSON api data, we'll not need to do the substring anymore. So, let's keep it like this for now.